packages/search/assets/js/containers/Search/Search.js

```diff
@@ -28,11 +27,8 @@ class Search extends React.Component<Props> {
     @action componentDidMount() {
         this.query = searchStore.query;
         this.indexName = searchStore.indexName;
-        indexStore.loadIndexes().then(action((indexes: Array<Index>) => {
-            this.indexes = indexes.reduce((indexesObject: Object, index) => {
-                indexesObject[index.indexName] = index;
-                return indexesObject;
-            }, {});
+        indexStore.loadIndexes().then(action((indexes: {[indexName: string]: Index}) => {
+            this.indexes = {...indexes};
```

Comment:
we should more talk about `indexResources` or `searchResources`:

```suggestion
        indexStore.loadIndexResources().then(action((indexResources: {[resourceKey: string]: IndexResource}) => {
            this.indexResources = {...indexResources};
```